.github/dependabot.yml

```diff
@@ -4,7 +4,20 @@ updates:
     directory: "/"
     open-pull-requests-limit: 5
     schedule:
-      interval: daily
+      interval: weekly
+    groups:
+      # create large PR upgrading multiple infrastructure dependencies in one shot,
+      # only include upstream dependencies that are stable and have somewhat
+      # regular releases which would be otherwise hard to manually manage.
+      common-golang-dependencies:
+        patterns:
+        - "cloud.google.com/*"
+        - "github.com/aws/aws-sdk-go/*"
+        - "github.com/minio/minio-go/*"
+        - "golang.org/x/*"
+        - "google.golang.org/*"
+        - "github.com/prometheus/*"
+        - "go.opentelemetry.io/*"
```

Comment:
this is largely ok, just a minor issue ...

otel updates can particularly conflict for people those who import kopia SDK along with k8s imports in the same project. I think k8s also has an otel dependency, that if mismatches versions with kopia, could have weird compile time issues

@julio-lopez WDYT ?

Reply:
that is a problem regardless of this PR, right?

Reply:
that is true 90% of the times. each update has been some manual effort so far

also, to be clear I don't think this is a kopia issue, just sdk inter dependency conflicts.

Reply:
Yes. However, having the updates for opentelemetry as separate PRs may make it easier to "downgrade" if needed. Having said that, it is probably the same  effort as (or even more than) creating a separate commit with the required dep downgrades.